scripts/harvest-skill-metadata/harvest.py

```diff
@@ -393,7 +398,8 @@ def generate_summary(github: Github, skill_entry: SkillEntry):
         'name': skill_entry.name,
         'github_username': skill_entry.author,
 
-        'title': title,
+        'title': title,                   # TODO: Remove in 19.08
+        'display_name': title,
```

Comment:
if title will be removed, perhaps the variable should be named `display_name` ?

Reply:
Let's keep this PR short and sweet.  There will be rework soon for YAML support